code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -118,6 +118,14 @@
 			log_say("RevenantTransmit: [key_name(user)]->[key_name(M)] : [msg]")
 			user << "<span class='revenboldnotice'>You transmit to [M]:</span> <span class='revennotice'>[msg]</span>"
 			M << "<span class='revenboldnotice'>An alien voice resonates from all around...</span> <span class='revennotice'>[msg]</span>"
+			for(var/mob/ded in dead_mob_list)
```

Comment:
Just do var/ded, you don't need to typecast it; it's in the dead mob list and you're not even using the typecasting